### PR TITLE
admin: set EAN import available on new document

### PR DESCRIPTION
**Pull request by jma**
* Adds package distribution build to the test script
* Delete deprecated SharedModule from the projects

Co-Authored-by: Olivier DOSSMANN <git@dossmann.net>
Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

### CODE REVIEW

File: projects/admin/src/app/record/custom-editor/libraries/library.component.spec.ts [hunk 1]
```diff
@@ -41,7 +41,6 @@ describe('LibraryComponent', () => {
         FormsModule,
         ReactiveFormsModule,
         CoreModule,
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/custom-editor/libraries/library.component.spec.ts]: Should be RecordModule for consistency with app.module.ts, but no matter.

File: projects/admin/src/app/app.module.ts [hunk 2]
```diff
@@ -44,6 +44,8 @@ import { ExceptionDatesEditComponent } from './record/custom-editor/libraries/ex
 import { CirculationPolicyComponent } from './record/custom-editor/circulation-settings/circulation-policy/circulation-policy.component';
 import { TranslateModule, TranslateLoader as BaseTranslateLoader } from '@ngx-translate/core';
 import { TranslateLoader } from './translate/loader/translate-loader';
+import { DocumentEditorComponent } from './document-editor/document-editor.component';
+import { Bootstrap4FrameworkModule } from 'angular6-json-schema-form';
```
Comment (sebastiendeleze) [inline projects/admin/src/app/app.module.ts]: Third party libraries should be imported before locale items

File: projects/admin/src/app/service/editor.service.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  constructor(
+    private http: HttpClient,
+    private recordService: RecordService
+  ) { }
+
+  /**
+   * Get record from rero-ils API
+   * @param ean EAN code
+   */
+  getRecordFromBNF(ean) {
```
Comment (sebastiendeleze) [inline projects/admin/src/app/service/editor.service.ts]: Add return type to method and @returns to doc block

File: projects/admin/src/app/translate/i18n/it.json [hunk 4]
```diff
@@ -1,9 +1,8 @@
 {
-  "Action": "Azione",
-  "Action done": "Azione realizzata",
+  "Action": "Azione", "Action done": "Azione realizzata",
```
Comment (AoNoOokami) [inline projects/admin/src/app/translate/i18n/it.json]: Should it be on two separated lines?

**Comment by AoNoOokami** [comment]
Should'n we use the new template for PR?

**Pull request merged**